core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss

```diff
@@ -46,6 +46,10 @@ $max-container-width: 1100px;
     font-weight: $font-weight-bold;
 }
 
+.fist-column {
```

Comment:
I think there is a typo here `first-column`

Reply:
Also I think we can do here:

```scss
:host div:first-child {}
```

Reply:
used `div:first-child`